.changeset/clean-comics-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus-dev-ui": patch
```

Comment:
Do we need a changeset for `\dev` stuff? Or is that a holdover from when it was in `\packages`?

Reply:
I think our tooling will yell if I don't make a changeset for this.  We could disable it, but 🤷

Reply:
Usually I wouldn't care about extra changesets, but `@khanacademy/perseus-dev-ui` isn't a think right? Like there's no `packages/perseus-dev-ui`. Seems like it'd confuse new contributors.

Reply:
Going to try to mark the `package.json` as `private: true` https://docs.npmjs.com/cli/v10/configuring-npm/package-json#private  

I think that'll stop the tooling from asking for a changeset in this "package".

Reply:
Ok. That worked.  We shouldn't be publishing this package anyways so I think that's the right call.